crypto/nextgen_crypto/src/ed25519.rs

```diff
@@ -39,6 +39,13 @@ use failure::prelude::*;
 use serde::{de, export, ser};
 use std::fmt;
 
+/// The length of the Ed25519PrivateKey
+pub const ED25519_PRIVATE_KEY_LENGTH: usize = ed25519_dalek::SECRET_KEY_LENGTH;
+/// The length of the Ed25519PublicKey
+pub const ED25519_PUBLIC_KEY_LENGTH: usize = ed25519_dalek::PUBLIC_KEY_LENGTH;
```

Comment:
Does `Ed25519PublicKey::length()` still make sense? If it can be removed I'm all for it.

Reply:
Good point. I prefer it stays for now, to tell you the truth, I would even consider `Signature::length()` for consistency. Let me handle this in an upcoming PR, I have some other ideas too, but first let's get rid of old api.